OpenRA.Game/Map/Map.cs

```diff
@@ -212,91 +61,20 @@ public class Map : IReadOnlyFileSystem, IDisposable
 
 		public readonly Dictionary<CPos, TerrainTile> ReplacedInvalidTerrainTiles = new Dictionary<CPos, TerrainTile>();
 
-		// Generated data
-		public readonly MapGrid Grid;
-		public IReadOnlyPackage Package { get; private set; }
-		public string Uid { get; private set; }
-
-		public Ruleset Rules { get; private set; }
-		public SequenceSet Sequences { get; private set; }
-
-		public bool InvalidCustomRules { get; private set; }
-		public Exception InvalidCustomRulesException { get; private set; }
-
-		/// <summary>
-		/// The top-left of the playable area in projected world coordinates
-		/// This is a hacky workaround for legacy functionality.  Do not use for new code.
-		/// </summary>
-		public WPos ProjectedTopLeft { get; private set; }
-
-		/// <summary>
-		/// The bottom-right of the playable area in projected world coordinates
-		/// This is a hacky workaround for legacy functionality.  Do not use for new code.
-		/// </summary>
-		public WPos ProjectedBottomRight { get; private set; }
-
-		public CellLayer<TerrainTile> Tiles { get; private set; }
-		public CellLayer<ResourceTile> Resources { get; private set; }
-		public CellLayer<byte> Height { get; private set; }
-		public CellLayer<byte> Ramp { get; private set; }
-		public CellLayer<byte> CustomTerrain { get; private set; }
-
-		public PPos[] ProjectedCells { get; private set; }
-		public CellRegion AllCells { get; private set; }
-		public List<CPos> AllEdgeCells { get; private set; }
-
-		public event Action<CPos> CellProjectionChanged;
+		public IReadOnlyPackage Package { get; protected set; }
+		public string Uid { get; protected set; }
+		public bool InvalidCustomRules { get; protected set; }
+		public Exception InvalidCustomRulesException { get; protected set; }
 
 		// Internal data
-		readonly ModData modData;
-		CellLayer<short> cachedTerrainIndexes;
-		bool initializedCellProjection;
-		CellLayer<PPos[]> cellProjection;
-		CellLayer<List<MPos>> inverseCellProjection;
-		CellLayer<byte> projectedHeight;
-		Rectangle projectionSafeBounds;
+#pragma warning disable IDE1006 // Naming Styles
+		protected readonly ModData modData;
+		protected Translation Translation;
+#pragma warning restore IDE1006 // Naming Styles
```

Comment:
From memory following this rule actually isn't possible because `modData` must be distinct from `ModData`, yet the naming rule does not allow lowercase 'm'. So the only way to follow the rule is to do something like `ModDataInstance`, which would be nastier than disabling the rule for those two lines imo.